src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bar {
+  position: relative;
+  margin-top: 20px;
```

Comment:
If you are going to give a distance between two forms, you should give a margin to a parent element (form), not to a child one